gwt-ol3-client/src/main/java/ol/MapOptions.java

```diff
@@ -32,49 +35,110 @@
 
     @JsProperty
-    public native String getTarget();
+    public native Collection<Control> getControls();
 
+	/**
+	 * @param pixelRatio The ratio between physical pixels and device-independent pixels (dips) on the device.
+	 * If <code>undefined</code> then it gets set by using <code>window.devicePixelRatio</code>.
+	 */
     @JsProperty
-    public native void setTarget(String target);
+    public native void setPixelRatio(Double pixelRatio);
 
     @JsProperty
-    public native void setLayers(Collection<Base> layers);
+    public native Double getPixelRatio();
+
+	/**
+	 * Interactions that are initially added to the map. If not specified,
+	 * ol.interaction.defaults() is used.
+	 *
+	 * @param interactions
+	 */
+	@JsProperty
+	public native void setInteractions(Collection<Interaction> interactions);
 
     @JsProperty
-    public native View getView();
+    public native void setInteractions(Interaction[] interactions);
 
     /**
-     * Controls initially added to the map.
-     *
-     * @param controls controls
+     * The element to listen to keyboard events on. This determines when the {@link ol.interaction.KeyboardPan} and {@link ol.interaction.KeyboardZoom} interactions trigger.
+     * For example, if this option is set to <code>document</code> the keyboard interactions will always trigger.
+     * If this option is not specified, the element the library listens to keyboard events on is the map target
+     * (i.e. the user-provided div for the map). If this is not <code>document</code> the target element needs to be focused for key events to be emitted,
+     * requiring that the target element has a <code>tabindex</code> attribute.
+     * 
+     * @param target The id of the element to listen keyboard events on.
      */
     @JsProperty
-    public native void setControls(Collection<Control> controls);
+    public native void setKeyboardEventTarget(String target);
 
     /**
-     * Set the map's view. No layer sources will be fetched unless this is
-     * specified at construction time or through ol.Map#setView.
-     *
-     * @param view {@link View}
+     * The element to listen to keyboard events on. This determines when the {@link ol.interaction.KeyboardPan} and {@link ol.interaction.KeyboardZoom} interactions trigger.
+     * For example, if this option is set to <code>document</code> the keyboard interactions will always trigger.
+     * If this option is not specified, the element the library listens to keyboard events on is the map target
+     * (i.e. the user-provided div for the map). If this is not <code>document</code> the target element needs to be focused for key events to be emitted,
+     * requiring that the target element has a <code>tabindex</code> attribute.
+     * 
+     * @param target The element to listen keyboard events on.
      */
     @JsProperty
-    public native void setView(View view);
+    public native void setKeyboardEventTarget(Element target);
```

Comment:
I tested setting **Document** (not Element) as keyboard event target and that works.